nvm.sh

```diff
@@ -2961,7 +2961,7 @@ nvm() {
           MANPATH=$(manpath)
         fi
         # Change current version
-        MANPATH="$(nvm_change_path "${MANPATH}" "/share/man" "${NVM_VERSION_DIR}")"
+        MANPATH=":$(nvm_change_path "${MANPATH}" "/share/man" "${NVM_VERSION_DIR}")"
```

Comment:
what happens if `MANPATH` already is prefixed by a colon, wouldn't this cause a double colon prefix?

Reply:
Yes it would cause a double colon. As per the manpage excerpt I posted, the determined (by the `manpath.config`) would then be inserted in between those colons and would therefore still be valid (and working).
A quick test with manually added double colons confirmed that it is still working. Of course we could make sure to add only one colon, but it has to be at least one to be working correctly.

Reply:
I tested it also with three colons, I still have working manpages again.
But if you can, please test on your machine quickly. I don't want to be the guy who says "On my machine it works" :)

Reply:
I don't personally use `man` much :-) how should I go about testing it locally?

Reply:
My error message without the fix is basically this:
```bash
$ man ls
No manual entry for ls
See 'man 7 undocumented' for help when manual pages are not available.
```
With the fix the normal manpage is shown. This is not isolated to the `ls` command but expands to (almost) everything.
Therefore I would suggest trying `man ls` with and without the fix and if you have a little bit of time, than the fix can be expanded with multiple colons to test these scenarios. 
In between the changes to `nvm.sh` you obviously need to start a clean terminal but other than that it should be pretty straight forward.

Reply:
hmm - on my Mac, this doesn't seem to occur. Do I have to have that manpath.config file present?

Reply:
That's interesting. I don't really know that much about the manpath on Macs, but I would suspect, as this is an issue on Ubuntu that the `nvm_change_path` function changes its behaviour on the systems. However I can not really confirm that (or really know what this function does).

Reply:
Does the fix change the behaviour of `man` in any way?

Reply:
it doesn't seem to have any impact at all - everything works fine with or without this change.

Reply:
Okay it shouldn't have an impact if the manpath has a colon in front so that is good to hear. I spoke with some colleagues and it seems to be an isolated ubuntu issue.

The problem is, I don't really know nvm that well, so I don't feel comfortable debuging the function on different systems. But this pull request is probably only viable as a quickfix.

Therefore I would like to leave it up to you if you want to merge it or just take it as an issue/ticket.
Thanks for the conversation! :)

Reply:
Okay talked some more about this with some colleagues and did some more detective work, it apparently is not nvm causing the issue, but puppet has a nasty bug in the Ubuntu lts package.